### ISSUE TITLE
Mask-level textual captions are not included in the public dataset.

### ISSUE THREAD

**Issue by businiao-zq** [zqcrafts/SkySense-O#22]
Thank you very much to the authors for their excellent work and for making the code and dataset publicly available. I noticed that the released dataset does not include mask-level textual captions. Will this part of the data be made publicly available in the future?

**Comment by zqcrafts** [comment]
There seems to be a misunderstanding. Our open-source dataset includes mask-text pairs in the skysa_dataset.jsonl file.

**Comment by businiao-zq** [comment]
<img width="930" height="819" alt="Image" src="https://github.com/user-attachments/assets/53903b9b-e583-4f46-bd81-a9b98c2bc6e1" />

<img width="699" height="520" alt="Image" src="https://github.com/user-attachments/assets/5700d6e9-cf34-482e-883c-70c7198cf44a" />

Thanks for the clarification.
I see the mask-level category tags (e.g., "car, railway track") in the file. 

However, I am looking for the **whole image captions** generated by GPT-4V as shown in **Figure 2** of your paper (e.g., "An elegant residential environment...").

Are these full image descriptions included in the dataset? I couldn't find them in the jsonl file.

**Comment by zqcrafts** [comment]
Image captions are part of the pre-training data, which we have not open-sourced. We only release high-quality fine-tuning data.

**Comment by businiao-zq** [comment]
Understood, thank you for the clarification. Thanks again for your great work!
